src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"messages.securitycraft:universalBlocModifier.cantModify": "\"",
+	"messages.securitycraft:universalOwnerChanger.noName": "Vous devez donner à cet élément le nom du nouveau propriétaire du bloc. Utilisez une enclume pour le renommer.",
+	"messages.securitycraft:universalOwnerChanger.cantChange": "Ce bloc ne peut pas contenir de propriétaire. Veuillez cliquer avec le bouton droit sur un bloc SecurityCraft approprié.",
+	"messages.securitycraft:universalOwnerChanger.notOwned": "Ce n’est pas votre bloc, vous ne pouvez pas changer de propriétaire.",
+	"messages.securitycraft:universalOwnerChanger.changed": "# est maintenant le nouveau propriétaire.",
+	"messages.securitycraft:universalOwnerChanger.noBlocClaiming": "Vous ne pouvez pas revendiquer des blocs sans propriétaire.",
+	"messages.securitycraft:irc.connected": "Le chat IRC n’est plus pris en charge à partir de la v1.8.3 de SecurityCraft. Si vous avez besoin d’aide, veuillez vous référer au canal #help sur SecurityCraft’s Discord!",
+	"messages.securitycraft:cageTrap.captured": "#player a été capturé à #location.",
+	"messages.securitycraft:frame.rightclick": "RClic droit avec un moniteur associé pour le voir.",
+	"messages.securitycraft:command.sc.usage": "Usage: /sc connect OU /sc disconnect OU /sc contact OU /sc resume",
+	"messages.securitycraft:naming.alreadyMatches": "Le nom de ce bloc correspond déjà ’#n’.",
+	"messages.securitycraft:naming.error": "Vous ne pouvez pas nommer ce bloc ’#n’, car le nom contient le caractère non valide ’#c’.",
+	"messages.securitycraft:thanks": "Merci d‘avoir utilisé SecurityCraft #!",
+	"messages.securitycraft:tip": "Conseil:",
+	"messages.securitycraft:tip.scHelp": "Tapez /sc help affichera le manuel SecurityCraft, qui affichera des informations d’aide pour les blocs / éléments SecurityCraft.",
+	"messages.securitycraft:tip.trello": "Consultez le tableau Trello pour SecurityCraft et signalez des bogues ou faites-nous des suggestions!",
+	"messages.securitycraft:tip.patreon": "Voulez-vous prendre en charge SecurityCraft?",
+	"messages.securitycraft:tip.discord": "Rejoignez notre serveur Discord pour discuter et recevoir des mises à jour et des annonces importantes concernant SecurityCraft!",
+	"messages.securitycraft:tip.scserver": "Le serveur officiel SecurityCraft est de retour ! Rejoignez-nous sur cette IP: 64.38.249.237",
+	"messages.securitycraft:universalKeyChanger.notKeypad": "Clic droit sur un clavier avec le changeur de code universel pour changer son mot de passe.",
+	"messages.securitycraft:universalKeyChanger.passcodeChanged": "Le code a été changé.",
+	"messages.securitycraft:passwordProtected.notSetUp": "Ce bloc n’a pas encore de mot de passe. Vous ne pouvez pas interagir avec.",
+	"messages.securitycraft:sentry.needsBlocBelow": "Les Sentinelles se placent sur un bloc!",
+	"messages.securitycraft:sentry.needsBlocsAround": "Assurez-vous que votre Sentinelle est entourée de blocs sur ses quatre côtés!",
```

Comment:
You accidentally removed the `k` from `Block` in the language keys here. They should be `messages.securitycraft:sentry.needsBlockBelow` and `messages.securitycraft:sentry.needsBlocksAround`.

Reply:
Still needs to be changed.